dashboard/test/controllers/courses_controller_test.rb

```diff
@@ -45,7 +45,7 @@ class CoursesQueryCountTests < ActionController::TestCase
 
     test_user_gets_response_for :index, response: :success, user: :user, queries: 4
 
-    test_user_gets_response_for :show, response: :success, user: :teacher, params: -> {{course_name: @unit_group_regular.name}}, queries: 11
+    test_user_gets_response_for :show, response: :success, user: :teacher, params: -> {{course_name: @unit_group_regular.name}}, queries: 14
```

Comment:
Since this test is not explicitly testing a cached state I just updated the query count. Still wondering why the validation is increasing query count here.